readiness-tool/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "chrome-ready-ness-tool",
+  "version": "10.3.1",
+  "description": "A tool for the sales team to better market AMP",
+  "author": "Alanna Walton <alwalton@google.com>",
+  "contributors": [
+    {
+      "name": "Alanna Walton",
+      "email": "alwalton@google.com"
+    }
+  ],
```

Comment:
Remove this for now. If there are multiple contributors later, they can be added in a separate CONTRIBUTERS.md file.